### PR TITLE
RA-807 Added label translations for Manage visit types

### PR DESCRIPTION
**Pull request by suthagar23**
# Description

The pages from the [Mange Visit types](https://github.com/suthagar23/openmrs-module-adminui/tree/ebd826450caf8119db314bead633d7155a679ee7/omod/src/main/webapp/pages/metadata/visits/visittypes/templates) need these translation labels for the Ref App. 

# Ticket 

Ticket : https://issues.openmrs.org/browse/RA-807

### CODE REVIEW

File: api/src/main/resources/messages.properties [hunk 1]
```diff
@@ -105,3 +105,17 @@ referenceapplication.configuremetadata=Configure Metadata
 
 referenceapplication.formentryapp.manageforms.label=Manage Forms
 
+general.auditInfo=Audit Info
```
Comment (wluyima) [inline api/src/main/resources/messages.properties]: Some of these might be already in core

Comment (suthagar23) [reply api/src/main/resources/messages.properties]: I couldn't find these in the Core. Let's see here : https://github.com/search?p=2&q=org%3Aopenmrs+general.auditInfo&type=Code

Am I missing anything here?

Comment (wluyima) [reply api/src/main/resources/messages.properties]: Most of these are already defined in the platform

Comment (suthagar23) [reply api/src/main/resources/messages.properties]: Yes, got it. So I just added required once here

**Pull request merged**